src/tools/tidy/src/style.rs

```diff
@@ -141,17 +141,27 @@ pub fn check(path: &Path, bad: &mut bool) {
     super::walk(path, &mut super::filter_dirs, &mut |entry, contents| {
         let file = entry.path();
         let filename = file.file_name().unwrap().to_string_lossy();
-        let extensions = [".rs", ".py", ".js", ".sh", ".c", ".cpp", ".h"];
+        let extensions = [".rs", ".py", ".js", ".sh", ".c", ".cpp", ".h", ".md"];
         if extensions.iter().all(|e| !filename.ends_with(e)) ||
            filename.starts_with(".#") {
             return
         }
 
+        if filename.ends_with(".md") &&
```

Comment:
And this seem like it's just wrong? We should style-check more than just error codes...

Reply:
And that's what we do. Or did I miss something? This condition is to opt out all md files that aren't error code long explanation (which is currently the case). All the other files are still checked.

Reply:
Why are we opting out non-error code markdown files? That seems like a strange distinction to make. The comment should also be updated; right now it's pretty misleading...

Reply:
We currently don't test them (lot of them are books and they have a lot of cases that break the linter) and the goal of this PR is clearly not to add checks for them. I'll update the comment quickly!

Reply:
Ah, I see. Since this PR first adds all markdown files, you want to then ignore many of them. Makes sense. An updated comment (and especially one looking towards the future, e.g., "eventually we'll likely want to check more files") would be great.